sentry/sentry.conf.example.py

```diff
@@ -291,3 +291,19 @@ def get_internal_network():
 
 # BITBUCKET_CONSUMER_KEY = 'YOUR_BITBUCKET_CONSUMER_KEY'
 # BITBUCKET_CONSUMER_SECRET = 'YOUR_BITBUCKET_CONSUMER_SECRET'
+
+##############################################
+# Suggested Fix Feature / OpenAI Integration #
+##############################################
+
+# See https://docs.sentry.io/product/issues/issue-details/ai-suggested-solution/
+# for more information about the feature. Make sure the OpenAI's privacy policy is
+# aligned with your company.
+
+# Set the OPENAI_API_KEY on the .env or .env.custom file with a valid
+# OpenAI API key to turn on the feature.
+OPENAI_API_KEY = env("OPENAI_API_KEY", "")
+
+if OPENAI_API_KEY and not OPENAI_API_KEY.isspace():
+  SENTRY_FEATURES["organizations:open-ai-suggestion"] = True
+  SENTRY_FEATURES["organizations:open-ai-suggestion-new-design"] = True
```

Comment:
As a side note:

You can actually just enable the `organizations:open-ai-suggestion` feature flag. Enabling both will gives you 2 section of suggested fix (before the stacktraces and after the breadcrumbs). But there's no way we can enable the `new-design` flag without enabling the first flag. Tried it before.

If we want some stability we can just enable only the `organizations:open-ai-suggestion` flag.

Reply:
Let's hold off on the `-new-design` flag for now since it seems to still be under development and not ready for prime time.

Reply:
Done